lbrynet/daemon/Daemon.py

```diff
@@ -1969,10 +1967,6 @@ def jsonrpc_publish(self, name, bid, metadata=None, file_path=None, fee=None, ti
         if bid <= 0.0:
             raise Exception("Invalid bid")
 
-        if bid >= self.session.wallet.get_balance():
-            raise InsufficientFundsError('Insufficient funds. ' \
```

Comment:
Same reason as the other bid check

Reply:
Also, this should be formatted as something like:
```python
raise InsufficientFundsError(
  'Insufficient funds. Make sure you have enough LBC to deposit.'
)
```
Or perhaps, if the message is constant it should just be built-in to the InsufficientFundsError. How many different messages do we need for this exception?